### ISSUE TITLE
Album styling breaks if Masonry view is disabled 

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtMediaWP/rtMedia#1520]
- keep masonry view disable check the albums styling it looks broken
path -> rtMedia settings -> Display settings -> MASONRY VIEW disabled

![image](https://user-images.githubusercontent.com/41000648/71477072-0cf11200-280e-11ea-9ffa-15182f26404a.png)

**Comment by pooja-muchandikar** [comment]
@dishitpala 

Tested with `pre-release` branch and album styling looks fine with Masonry view enabled/disabled both :heavy_check_mark: 

```
Test Environment
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2 ( with nouveau and legacy template )
rtMedia: v4.5.11 pre-release branch ( with Masonry view disable and enabled )
Theme: 2019, 2020
Browser: Chrome, Firefox and Safari
```

**Comment by pooja-muchandikar** [comment]
Tested with `cheery-pick/4.5.12` branch and the fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and fix added works as expected. 

```
Test Environment
WordPress: v5.3.2
BuddyPress: v5.1.2 ( with nouveau and legacy template )
rtMedia: v4.5.11 develop branch ( with Masonry view disable and enabled )
Theme: 2019, 2020
Browser: Chrome, Firefox and Safari
```

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix added works as expected.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.0`
